lib/graphql/transfer.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  CreateTransferInput,
+  Transfer as TransferEntry,
+  BatchTransfer
+} from "./schema";
+import { Model } from "./model";
+import { FetchOptions } from "./types";
+import { ResultPage } from "./resultPage";
+
+export class Transfer extends Model<TransferEntry> {
+  async createOne(transfer: CreateTransferInput): Promise<TransferEntry> {
+    const query = `mutation {
+      createTransfer(
+        transfer: { 
+          recipient: "${transfer.recipient}", 
+          iban: "${transfer.iban}", 
+          amount: ${transfer.amount}, 
+          e2eId: "${transfer.e2eId}", 
+          purpose: "${transfer.purpose}" 
+        }
+      ) {
+        id
+        status
+        amount
+        note
+        iban
+        e2eId
+      }
+    }`;
+
+    const result = await this.client.rawQuery(query);
+    return result.createTransfer;
+  }
+
+  async confirmOne(
+    confirmationId: string,
+    authorizationToken: string
+  ): Promise<TransferEntry> {
+    const query = `mutation {
+      confirmTransfer(
+        transferId: "${confirmationId}"
+        authorizationToken: "${authorizationToken}"
+      ) {
+        status
+        amount
+        note
+        id
+        recipient
+        iban
+        e2eId
+      }
+    }`;
+
+    const result = await this.client.rawQuery(query);
+    return result.confirmTransfer;
+  }
+
+  async createMany(transfers: Array<CreateTransferInput>): Promise<string> {
+    const query = `mutation createTransfers($transfers: [CreateTransferInput!]!) {
+      createTransfers(transfers: $transfers) {
+        confirmationId
+      }
+    }`;
+
+    const result = await this.client.rawQuery(query, { transfers });
+    return result.createTransfers.confirmationId;
+  }
+
+  async confirmMany(
+    confirmationId: string,
+    authorizationToken: string
+  ): Promise<BatchTransfer> {
+    const query = `mutation {
+      confirmTransfers(
+        confirmationId: "${confirmationId}"
+        authorizationToken: "${authorizationToken}"
+      ) {
```

Comment:
I think it would be more secure to pass `variables` alongside `query` instead of interpolating values it JS (the same that you do for `createMany`). It should prevent possible injections.